app/guides/branching.md

```diff
@@ -8,24 +8,118 @@ tags:
 
 To do this, add some JavaScript to your `routes.js` file.
 
-Here is an example:
+## Branching on radio button answers
+
+Send users to different pages based on their radio button selection:
+
+```js
+router.post('/eligibility-answer', function (req, res) {
+  const data = req.session.data
+  const eligible = data.isEligible
+
+  if (eligible == "Yes") {
+    res.redirect('/next-question')
+  }
+  else {
+    res.redirect('/not-eligible')
+  }
```

Comment:
```suggestion
  if (eligible == "Yes") {
    res.redirect('/next-question')
  }
  elseif (eligible == "No") {
    res.redirect('/not-eligible')
  }
  else {
    res.redirect('/eligibility')
  }
```

I feel we should be testing each option explicitly (and teaching users to do that) rather than defaulting things. With the existing pattern, they can get to the check-answers page and be missing data otherwise.